### PR TITLE
[darktable-cli] document new options

### PR DESCRIPTION
**Pull request by johnny-bit**
I hope I did it right :) I haven't checked how it looks since building hugo on my system seemed weird. and gentoo only has 0.69 while newest seems to be 0.75

### CODE REVIEW

File: content/special-topics/program-invocation/darktable-cli.md [hunk 1]
```diff
@@ -67,9 +74,30 @@ _where `<FORMAT>` is the name of the selected output format.__
 `--style-overwrite`
 : The specified style overwrites the history stack instead of being appended to it.
 
+`--apply-custom-presets <0|1|false|true>`
+: Whether to load `data.db` which contains presets and styles. Disabling this option allows to run multiple instances of `darktable-cli` at the cost of inability to use `--style` option. Defaults to true.
+
+`--out-ext <extension>`
+: Set the output extension to use. If specified takes precedence over `<output file>`. Defaults to extracted from `<output file>` or `jpg` in case `<output folder>` is specified.
```
Comment (elstoc) [inline content/special-topics/program-invocation/darktable-cli.md:81]: This isn't clear. Do you mean "By default this is extracted from `<output file>`. Defaults to `jpg` if `<output folder>` is specified"?

File: content/special-topics/program-invocation/darktable-cli.md [hunk 2]
```diff
@@ -67,9 +74,30 @@ _where `<FORMAT>` is the name of the selected output format.__
...
+
+`--import <file or dir>`
+: Specify input file or folder, can be used multiple times. Do not combine this option with `<input file or folder>`.
```
Comment (elstoc) [inline content/special-topics/program-invocation/darktable-cli.md:84]: What happens if you try to? Does it fall in a heap? If so, may be better to say "This option cannot be combined with..."

File: content/special-topics/program-invocation/darktable-cli.md [hunk 3]
```diff
@@ -67,9 +74,30 @@ _where `<FORMAT>` is the name of the selected output format.__
...
+
+`--icc-type <type>`
+: Specify ICC type. Defaults to NONE. Use `--help icc-type` for list of supported types.
```
Comment (elstoc) [inline content/special-topics/program-invocation/darktable-cli.md:87]: Is this better with "ICC *profile* type"? Similarly below.

File: content/special-topics/program-invocation/darktable-cli.md [hunk 4]
```diff
@@ -67,9 +74,30 @@ _where `<FORMAT>` is the name of the selected output format.__
...
+
+`--icc-file <file>` 
+: Specify ICC filename. Defaults to empty filename.
+
+`--icc-intent <intent>` 
+: Specify ICC intent, Defaults to LAST. Use `--help icc-intent` for list of supported intents.
```
Comment (elstoc) [inline content/special-topics/program-invocation/darktable-cli.md:93]: What does LAST mean here? Perhaps worth specifying given that it's the default.

**Review by elstoc — COMMENTED** [review]
Thanks @johnny-bit. A few queries/clarifications but looks good. There are a couple of other really minor things that I'll pick up with a copy-edit after merging.

**Comment by johnny-bit** [comment]
Done :)

**Comment by elstoc** [comment]
> Done :)

Maybe... What does "Defaults to image specified" mean in the ICC options?

**Comment by johnny-bit** [comment]
same as `export module` profile & intent `image specified`?

**Comment by elstoc** [comment]
ok probably just needs some quotation marks to clarify. I'll merge now and pick it up in my copy-edit. Thanks again.

**Pull request merged**